package/MDAnalysis/coordinates/TRJ.py

```diff
@@ -1130,6 +1228,11 @@ class NCDFPicklable(scipy.io.netcdf.netcdf_file):
     mmap : None or bool, optional
         Whether to mmap `filename` when reading. True when `filename`
         is a file name, False when `filename` is a file-like object.
+    version : {1, 2}, optional
```

Comment:
Not clear what this is and why it's needed. Link to other docs available?

Reply:
The sentence is ambiguous: Could be read as if it is putting "netcdf file version" into "read/write" mode.

Reply:
"Sets the netcdf version. 1 is classic, 2 is the 64-bit offset format"? (also I notice the 65 bit typo)

It's less descriptive but as mentioned in the other comment we can link to the scipy docs for more information? (not that they are more descriptive with this parameter...)

Reply:
Linking to scipy docs is good.

Should we even allow setting the format? Doesn't ncdf always use version=2? According to http://ambermd.org/netcdf/nctraj.xhtml

> **Trajectory/restart files shall use 64 bit offsets**
> This can be accomplished by setting a flag during file creation; refer to API docs for details.

Reply:
So interestingly we don't set version on read, but we do on write. I need to double check if it defaults to 2.... otherwise we got a potential bug on our hands.

Reply:
Ok yeah so the version_byte is just picked up from the file on read, so it _shouldn't_ be an issue that we don't set the flag when reading. Either way we throw an error if the version_byte is wrong, so that's fine.